src/test/HostActivationTests/GivenThatICareAboutPortableAppActivation.cs

```diff
@@ -125,6 +125,37 @@ public void Muxer_Exec_activation_of_Build_Output_Portable_DLL_with_DepsJson_Loc
         }
 
         [Fact]
+        public void Muxer_Activation_With_Templated_AdditionalProbingPath_Succeeds()
+        {
+            var fixture = PreviouslyBuiltAndRestoredPortableTestProjectFixture
+                .Copy();
+            
+            var store_path = CreateAStrore(fixture);
+            var dotnet = fixture.BuiltDotnet;
+            var appDll = fixture.TestProject.AppDll;
+
+            var destRuntimeDevConfig = fixture.TestProject.RuntimeDevConfigJson;
+            if (File.Exists(destRuntimeDevConfig))
+            {
+                File.Delete(destRuntimeDevConfig);
+            }
+
+            var additionalProbingPath = store_path + "/{arch}/{tfm}";
+
+            dotnet.Exec(
+                    "exec",
+                    "--additionalprobingpath", additionalProbingPath,
+                    appDll)
+                .CaptureStdErr()
+                .CaptureStdOut()
+                .Execute()
+                .Should()
+                .Pass()
+                .And
+                .HaveStdOutContaining("Hello World");
```

Comment:
Should we verify that the Newtonsoft.Json assembly is loaded from the `store_path`?